components/Settings.js

```diff
@@ -69,6 +70,14 @@ class Settings extends React.Component {
             onChange={this.props.onChange.bind(null, 'widthAdjustment')}
           />
           <Collapse label="Advanced">
+            <Slider
+              label="Line height"
+              value={this.props.lineHeight}
+              minValue={90}
+              maxValue={250}
+              usePercentage={true}
+              onChange={this.props.onChange.bind(null, 'lineHeight')}
+            />
```

Comment:
Happy to move this, but I put it to the top so it's closest to the font-size control while still being in the 'advanced' settings.